### ISSUE TITLE
<select> element does not parse background-color change

### ISSUE THREAD

**Issue by otamam818** [tauri-apps/tauri#5766]
(__Note__: This is my first time posting a GitHub issue, apologies if I missed something and thank you for your time!)

### Describe the Bug
The `<select>` element does not follow the scss description.

In the following image, the `<input>` textbox below the `Price` label is properly styled, but the style does not apply to the `<select>` tag beside that. 
![image](https://user-images.githubusercontent.com/68072081/205699249-7d1bb2b4-21c9-4328-a284-a0f9c84b9de9.png)

The same CSS styling I did reproduced my desired style in the web browser just fine
![image](https://user-images.githubusercontent.com/68072081/205701488-54c2961e-d9fb-4df1-a810-d7927405266d.png)

### Reproduction

__HTML__
```HTML
<select>
  <option value="Hello"> Hello </option>
</select>
```

__CSS__
```CSS
select {
  background-color: darkred;
}
```

### Expected behavior
Background for `<select>` element renders like in the second screenshot

### Platform and versions
```
npm run tauri info

> app_rs@0.0.0 tauri
> tauri "info"


Environment
  › OS: Fedora 36.0.0 X64
  › Node.js: 16.14.0
  › npm: 8.3.1
  › pnpm: Not installed!
  › yarn: 1.22.17
  › rustup: 1.25.1
  › rustc: 1.63.0
  › cargo: 1.63.0
  › Rust toolchain: stable-x86_64-unknown-linux-gnu 

Packages
  › @tauri-apps/cli [NPM]: 1.2.1
  › @tauri-apps/api [NPM]: 1.2.0
  › tauri [RUST]: 1.2.1,
  › tauri-build [RUST]: 1.2.1,
  › tao [RUST]: 0.15.6,
  › wry [RUST]: 0.22.5,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../dist
  › devPath: http://localhost:1420/
  › framework: React

App directory structure
  ├─ src-tauri
  ├─ .vscode
  ├─ src
  ├─ public
  └─ node_modules
```
__package.json___
```json
{
  "name": "app_rs",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview",
    "tauri": "tauri"
  },
  "dependencies": {
    "@fortawesome/fontawesome-svg-core": "^6.2.1",
    "@fortawesome/free-regular-svg-icons": "^6.2.1",
    "@fortawesome/free-solid-svg-icons": "^6.2.1",
    "@fortawesome/react-fontawesome": "^0.2.0",
    "@iconscout/react-unicons": "^1.1.6",
    "@tauri-apps/api": "^1.1.0",
    "babel-plugin-macros": "^3.1.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-feather": "^2.0.10",
    "sass": "^1.56.1"
  },
  "devDependencies": {
    "@tauri-apps/cli": "^1.1.0",
    "@types/react": "^18.0.15",
    "@types/react-dom": "^18.0.6",
    "@vitejs/plugin-react": "^2.0.0",
    "vite": "^3.0.2"
  }
}
```

__tauri.conf.json__
```json
{
  "build": {
    "beforeDevCommand": "npm run dev",
    "beforeBuildCommand": "npm run build",
    "devPath": "http://localhost:1420",
    "distDir": "../dist",
    "withGlobalTauri": false
  },
  "package": {
    "productName": "app_rs",
    "version": "0.0.0"
  },
  "tauri": {
    "allowlist": {
      "dialog" : { "all" : true },
      "all": true
    },
    "bundle": {
      "active": true,
      "category": "DeveloperTool",
      "copyright": "",
      "deb": {
        "depends": []
      },
      "externalBin": [],
      "icon": [
        "icons/32x32.png",
        "icons/128x128.png",
        "icons/128x128@2x.png",
        "icons/icon.icns",
        "icons/icon.ico"
      ],
      "identifier": "com.tauri.dev",
      "longDescription": "",
      "macOS": {
        "entitlements": null,
        "exceptionDomain": "",
        "frameworks": [],
        "providerShortName": null,
        "signingIdentity": null
      },
      "resources": [],
      "shortDescription": "",
      "targets": "all",
      "windows": {
        "certificateThumbprint": null,
        "digestAlgorithm": "sha256",
        "timestampUrl": ""
      }
    },
    "security": {
      "csp": null
    },
    "updater": {
      "active": false
    },
    "windows": [
      {
        "fullscreen": false,
        "height": 600,
        "resizable": true,
        "title": "app_rs",
        "width": 800
      }
    ]
  }
}
```

### Stack Trace
```
No error/warning message.
```

**Comment by amrbashir** [comment]
This is not a specific tauri issuee, but rather a common css problem, please see this article https://moderncss.dev/custom-select-styles-with-pure-css/.

tldr: the magic is in adding `appearance: none`

**Comment by otamam818** [comment]
Ahh, I see. I'll try it out soon. Thanks 😁
